sdk/cosmos/azure-cosmos/azure/cosmos/_routing/collection_routing_map.py

```diff
@@ -120,10 +116,8 @@ def get_range_by_partition_key_range_id(self, partition_key_range_id):
     def get_overlapping_ranges(self, provided_partition_key_ranges):
         """Gets the partition key ranges overlapping the provided ranges
 
-        :param list provided_partition_key_ranges:
-            List of partition key ranges.
-        :return:
-            List of partition key ranges, where each is a dict.
+        :param list provided_partition_key_ranges:cList of partition key ranges.
```

Comment:
I mischievous "c"

Reply:
fixed in https://github.com/Azure/azure-sdk-for-python/pull/8607/commits/c14bb0d8adcb5e63ac1d6a2fab3819ccca50707e